test/ui-2/test/DropDownButtonStep.test.js

```diff
@@ -0,0 +1,20 @@
+const timeout = process.env.SLOWMO ? 30000 : 10000;
+const fs = require('fs');
+beforeAll(async () => {
+  path = fs.realpathSync('file://../examples/index.html');
+  await page.goto('file://' + path, {waitUntil: 'domcontentloaded'});
+});
+describe('add step dropdown', ()=>{
+  test('Click Button', async()=>{
+    await page.waitForSelector('.step');
+    const Length = await page.evaluate(() => document.querySelectorAll('.step').length);
+    await page.click('[data-value=\'brightness\']');
```

Comment:
Choose different value not present in radio group

Reply:
you also need to add $(".selectize-input").click() before this.

Reply:
Umm @keshav234156 why are we choosing value not present there